### PR TITLE
Avoid ESM-related issues

### PR DESCRIPTION
**Pull request by LitoMore**
Resolve https://github.com/vadimdemedes/ink/issues/435

Related to https://github.com/sindresorhus/meow/issues/172#issuecomment-833402476

Target some modules to its CommonJS module version to avoid some ESM-related issues.

### CODE REVIEW

**Comment by NSExceptional** [comment]
Yo @vadimdemedes, please merge this for all us typescript users :)

**Comment by LitoMore** [comment]
@sindresorhus Could you help? Do you have write access to this repo?

File: index.js [hunk 1]
```diff
@@ -108,19 +108,19 @@ module.exports = () => {
 			task: async () => {
 				await execa('npm', [
 					'install',
-					'meow',
-					'ink',
+					'meow@9',
+					'ink@3',
```
Comment (vadimdemedes) [inline index.js:112]: Since there's no 4.x version of Ink, do we need `@3` version range?

Comment (LitoMore) [reply index.js:112]: For avoiding ESM-related issues in the future, we would better limit its version.

**Comment by vadimdemedes** [comment]
Thank you!

**Comment by vadimdemedes** [comment]
Released in 2.1.1.

**Pull request merged**